pkg/validation/spec/info.go

```diff
@@ -53,6 +53,10 @@ func (e Extensions) GetBool(key string) (bool, bool) {
 // GetStringSlice gets a string value from the extensions
 func (e Extensions) GetStringSlice(key string) ([]string, bool) {
 	if v, ok := e[strings.ToLower(key)]; ok {
+		if alreadyStringSlice, ok := v.([]string); ok {
```

Comment:
xref https://github.com/kubernetes/kube-openapi/pull/413#issuecomment-1636119257

Reply:
removed this bit